src/run/mod.rs

```diff
@@ -17,12 +23,17 @@ pub fn run(args: &RunArgs) -> anyhow::Result<()> {
     if let Some(RunSubcommands::Web(web_args)) = &args.subcommand {
         ensure_web_setup()?;
 
+        let metadata = cargo::metadata::metadata_with_args(["--no-deps"])?;
+
         // If targeting the web, run a web server with the WASM build
         println!("Building for WASM...");
         cargo::build::command().args(cargo_args).ensure_status()?;
 
         println!("Bundling for the web...");
```

Comment:
These `println!()` messages could be clearer. When I first read them, I though they said the exact same thing!

```suggestion
        println!("Building project as WASM...");
        cargo::build::command().args(cargo_args).ensure_status()?;

        println!("Bundling WASM for the web...");
```

You may have better wording than that. My goal is to make the messages _clearer_ and _less alike_.

Reply:
I adjusted the messages, let me know if they work for you now!

Reply:
Yup, I like them better. Thanks!